addons/stock/wizard/stock_picking_responsible.py

```diff
@@ -1,23 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import _, fields, models
-from odoo.exceptions import UserError
-
-
-class StockPickingResponsible(models.TransientModel):
-    _name = 'stock.picking.responsible'
-    _description = 'Assign Responsible'
-
-    user_id = fields.Many2one(
-        'res.users', 'Responsible',
-        domain=lambda self: [('groups_id', 'in', self.env.ref('stock.group_stock_user').id)],
-        default=lambda self: self.env.user)
-
-    def assign_responsible(self):
```

Comment:
in this commit, i would remove only the views/data but keep the wizard

it's too late to change the model :(
add a comment # REMOVE ME IN 13.1 somewhere in the python file